mm2src/coins/qrc20.rs

```diff
@@ -994,6 +995,10 @@ impl MmCoin for Qrc20Coin {
         Box::new(qrc20_withdraw(self.clone(), req).boxed().compat())
     }
 
+    fn get_raw_transaction(&self, req: RawTransactionRequest) -> RawTransactionFut {
+        Box::new(qrc20_get_raw_transaction(self.clone(), req).boxed().compat())
```

Comment:
Qrc20Coin implements all the required traits, please use `utxo_common::get_raw_transaction` instead.